### PR TITLE
add Features to Media Module

### PR DESCRIPTION
**Pull request by ShivanshGairola**
Targets #2599

### Description
Added Features to Media Module

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Media/Resources/views/media/index.blade.php [hunk 1]
```diff
@@ -78,14 +71,13 @@
...
                         <img src="{{ asset('storage/media/'.$row->file_url) }}" class="card-img-top img-fluid ">
                     @endif
                 </a>
                 <div class="card-body">
-                    <p>
-                        <td class="">
-                            <img src="{{auth()->user()->avatar}}" alt="{{auth()->user()->name}}" class="w-25 h-25 rounded-circle" data-toggle="tooltip" data-placement="top" title="{{auth()->user()->name}}">
+                    <p>Uploaded By
+                        <td class=""> <img src="{{auth()->user()->avatar}}" alt="{{auth()->user()->name}}" class="w-25 h-25 rounded-circle" data-toggle="tooltip" data-placement="top" title="{{auth()->user()->name}}">
                         </td>
```
Comment (GauravGusain98) [inline Modules/Media/Resources/views/media/index.blade.php:81]: Why you have used `<td>` tag when there is no table. Also remove the class attribute if there is no class we are using

File: Modules/Media/Resources/views/media/index.blade.php [hunk 2]
```diff
@@ -78,14 +71,13 @@
                             <source src="{{asset('storage/media/'.$row->file_url)}}" type="video/mp4">
                         </video>
                     @endif
-                    @if(pathinfo($row->file_url, PATHINFO_EXTENSION) == 'jpg' || pathinfo($row->file_url, PATHINFO_EXTENSION) == 'jpeg' || pathinfo($row->file_url, PATHINFO_EXTENSION) == 'png' )
+                    @if(pathinfo($row->file_url, PATHINFO_EXTENSION) == 'jpg' || pathinfo($row->file_url, PATHINFO_EXTENSION) == 'jpeg' || pathinfo($row->file_url, PATHINFO_EXTENSION) == 'PNG' )
```
Comment (GauravGusain98) [inline Modules/Media/Resources/views/media/index.blade.php:74]: Why only `PNG` is in uppercase, while `jpg` and `jpeg` are lowercase? Can you confirm if the syntax is correct

File: Modules/Media/Http/Controllers/MediaController.php [hunk 3]
```diff
@@ -32,22 +32,26 @@ public function index()
...
-        );
+            foreach ($files as $file) {
+                $validated = $request->validated();
+                $path = config('media.path');
+                $fileName = $file->getClientOriginalName();
+                $file->move($path, $fileName);
+                array_push($names, $fileName);
 
-        $postData = [
+                $postData = [
```
Comment (GauravGusain98) [inline Modules/Media/Http/Controllers/MediaController.php:46]: Please correct the indentation

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments. The feedbacks mostly include formatting issues. I'll do another round of review once these issues are fixed. Will also need some help in the testing of the workflow on local

File: Modules/Media/Resources/views/media/index.blade.php (hunk 2 above)
Comment (ShivanshGairola) [reply Modules/Media/Resources/views/media/index.blade.php:74]: Sir png image have extension in uppercase only.thats why used in upper case

File: Modules/Media/Http/Controllers/MediaController.php (hunk 3 above)
Comment (GauravGusain98) [reply Modules/Media/Http/Controllers/MediaController.php:46]: This is not implemented

Comment (ShivanshGairola) [reply Modules/Media/Http/Controllers/MediaController.php:46]: After implementing it failing in CI check.It is showing larstan error

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
- The text of this button is not updated.
    image
- The modal title and button text is also not updated
    image
- On Saving the media the image is not getting saved.

File: Modules/Media/Http/Controllers/MediaController.php (hunk 3 above)
Comment (GauravGusain98) [reply Modules/Media/Http/Controllers/MediaController.php:46]: ok

**Comment by GauravGusain98** [comment]
The image is not getting saved and is not getting displayed on the page. Please test it once in your system.

**Pull request closed without merging**